docker-compose.yml

```diff
@@ -38,6 +38,10 @@ services:
   solr:
     image: solr:7.5
     command: solr -c -f -z zoo1:2181
+    environment:
+      - OOM=script
+      - VIRTUAL_PORT=8983
+      - VIRTUAL_HOST=solr.hyku.docker
```

Comment:
Interesting.  Totally an edge case right now, but I wonder if we need to add a reserved name check for creating tenants to avoid collisions with pre-existing subdomains that were manually created like this one.

Reply:
good point. I'll open a separate PR for it